docs/gds-supported-platform/per-namespace-gateway-opt-in.md

```diff
@@ -0,0 +1,49 @@
+# Per-namespace istio gateways
+
+We are migrating from using a single `ingressgateway` in `istio-system` to an `ingressgateway` in each namespace. At present this requires an "opt-in" via `-cluster-config` values settings to enable.
+
+For example, from `tech-ops-cluster-config`'s `sandbox-values`:
```

Comment:
What do you do with this code?

Reply:
Nothing...it's scene-setting. Starting from this; add a) to simply opt-in with default; add b) to opt-in specifying additional ports.
Is there a better way to present this?

Reply:
I've restructured this document following a chat with @philandstuff. It's now based on the vision (no opt-in) with a section on "how to migrate / opt in". Hopefully that improves things.